build.sh

```diff
@@ -176,7 +176,7 @@ BuildLibevent()
     if [ $? -ne 0 ];then
         exit 1
     fi    
-    make
+    make -j4
```

Comment:
Let's define a variable to set the num of cpus like:
```shell
NUM_CPUS=4
...

make -j $NUM_CPUS

So that if some one's machine has 16 cores, he/she can specify this simply.

```

Reply:
good